netlify/netlify-cms
# Login workflow

**\- Summary**

This PR improves the login/logout flow by displaying a currently logged in userpic in the AppHeader + allowing to log out.

Closes #100

**\- Test plan**
1. Log in with GitHub
2. Click on the user pic in the right top corner
3. Click Logout
4. You'll see the login screen again

Unfortunately re-login introduceses some React errors. See #136 

**\- Description for the changelog**

`[feat](login) Added userpic with a logout action in the dropdown.`
